<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mserialport</class>
 <widget class="QWidget" name="Mserialport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mserialport</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="2" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_5" rowstretch="0,3,1,0">
      <item row="2" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>发送</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="2" column="0">
          <widget class="QPushButton" name="Button_send">
           <property name="text">
            <string>点击发送</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPlainTextEdit" name="ptesend"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="le">
        <property name="text">
         <string>   version: llx v0.0   1,设置串口参数    2,打开串口     3,设置灯带长度与每米个数    4灯带模式选择   5选择颜色   6,点击发送</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>接收</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="3,0,0" columnstretch="3,1">
         <item row="0" column="0" rowspan="2">
          <widget class="QPlainTextEdit" name="ptereceive"/>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="1,0,0,0">
            <item row="5" column="3">
             <widget class="QComboBox" name="comboBox_Check">
              <item>
               <property name="text">
                <string>无</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>奇校验</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>偶校验</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QComboBox" name="comboBox_Portname">
              <item>
               <property name="text">
                <string>COM1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM3</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM4</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>COM5</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>校验位</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="14" column="3">
             <widget class="QLineEdit" name="comboBox_Numberset"/>
            </item>
            <item row="4" column="3">
             <widget class="QComboBox" name="comboBox_Data">
              <item>
               <property name="text">
                <string>8</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>7</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>6</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>5</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="6" column="3">
             <widget class="QComboBox" name="comboBox_Stop">
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>1.5</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="3">
             <widget class="QComboBox" name="comboBox_Baud">
              <item>
               <property name="text">
                <string>9600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>19200</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>38400</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>57600</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>115200</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>数据位</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>串口名</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>停止位</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>波特率</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="12" column="3">
             <widget class="QLineEdit" name="comboBox_Lengthset"/>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>灯带总长度/m</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QPushButton" name="Button_open">
              <property name="text">
               <string>打开串口</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>16进制/ASCII</string>
              </property>
             </widget>
            </item>
            <item row="8" column="3">
             <widget class="QPushButton" name="Button_close">
              <property name="text">
               <string>关闭串口</string>
              </property>
             </widget>
            </item>
            <item row="14" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>每米灯个数</string>
              </property>
             </widget>
            </item>
            <item row="15" column="0">
             <widget class="QPushButton" name="Button_colorselect">
              <property name="text">
               <string>选择颜色</string>
              </property>
             </widget>
            </item>
            <item row="7" column="3">
             <widget class="QComboBox" name="comboBox_Fontset">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="editable">
               <bool>false</bool>
              </property>
              <item>
               <property name="text">
                <string>Hex</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>ASCII</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="15" column="3">
             <widget class="QComboBox" name="comboBox_Modelselect">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <item>
               <property name="text">
                <string>颜色控制</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>呼吸灯</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>流水灯</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>跑马灯</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="16" column="3">
             <widget class="QLineEdit" name="lineEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="16" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>CRC校验结果(0x)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>格式:帧头:0x48,命令码:0x06,数据长度:0x000C(12),数据域(模式选择,亮度,色温,R,G,B,长度,个数),CRC</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="Button_clear">
           <property name="text">
            <string>清除接收</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
